components/ai_chat/browser/engine/remote_completion_client.cc

```diff
@@ -101,9 +106,16 @@ const GURL GetEndpointBaseUrl() {
 
 }  // namespace
 
-AIChatAPI::AIChatAPI(
+// static
+std::string RemoteCompletionClient::GetHumanPromptSegment() {
+  return base::StrCat({"\n\n", kHumanPrompt, " "});
```

Comment:
@LorenzoMinto @nvonpentz is this meant to apply as a stop sequence to both llama2 and claude? I thought it wasn't used for Llama2 but we're still adding it as a stop sequence for all llama2 calls and the only stop sequence for llama2 conversation calls (the question generation call adds an additional stop sequence).

Reply:
Yes, this shouldn't be added as a stop sequence to llama as it doesn't use it. It may not affect the generation much as that sequence is generally unlikely, but we should probably keep this Claude specific.

Reply:
done